libHttpClient.props

```diff
@@ -46,6 +46,7 @@
     <HCRoot Condition="'$(HCRoot)'==''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.root))\</HCRoot>
     <HCBuildRoot>$(HCRoot)\Build</HCBuildRoot>
     <libHttpClientInclude>$(HCRoot)\Include\</libHttpClientInclude>
+    <zlibInclude>$(HCRoot)\External\zlib\</zlibInclude>
```

Comment:
This will add Zlib headers to client's include path. I think that's not what you want.  This probably belongs in libHttpClient.Zlib.vcxitems

Reply:
Fixed, I moved it to libHttpClient.Zlib.vcxitems